dockers/docker-snmp-sv2/snmpd.conf.j2

```diff
@@ -118,6 +118,7 @@ load   12 10 5
 #
                                            #  Run as an AgentX master agent
 master          agentx
+agentxsocket    tcp:localhost:3161
```

Comment:
>agentxsocket [](start = 0, length = 12)

How to test this feature? Can you add a test case in sonic-mgmt?

Reply:
you need an agentx client to be able to test this feature: we can provide an end to end test by using the bgp daemon as agentx client and doing an snmp request to a RFC1657 OID

Reply:
Not sure why we need an agentx client to test. I guess you can add some queries into existing sonic-mgmt snmp test.

Reply:
yes I'm looking into it right now, I'm checking to see if the ansible snmp_facts module queries the right OIDs which would make the test trivial.

Reply:
https://github.com/Azure/sonic-mgmt/blob/master/ansible/library/snmp_facts.py does not support RFC1657 at this time, we could patch it to add a subset of OIDs but its going to take some time. I suggest we add it to the issues backlog of sonic-mgmt and we can pick it up

Reply:
If you could not provide a sonic-mgmt test case right now, please add some manual test instruction here (with steps and sample output). We could not accept feature without test.

---
In reply to: [293172354](https://github.com/Azure/sonic-buildimage/pull/2981#discussion_r293172354) [](ancestors = 293172354)

Reply:
OK will add a link to look at the docker-fpm-frr snmp.conf file and put the test protocol there

Reply:
Added the info to both files